glide.lock

```diff
@@ -70,8 +70,6 @@ imports:
   version: 8f6c67f0228a4a6a962008d6c3d1e694e4532e71
 - name: github.com/go-utils/uslice
```

Comment:
We can get rid of this too:
https://github.com/chappjc/dcrstakepool/commit/71a30638d7d5ded1856a0b0bfc9625c3e80204ba